utils/obsproc/rtofs/RTOFSDataFile.cc

```diff
@@ -0,0 +1,114 @@
+#include "RTOFSDataFile.h"
+
+#include <ctime>
+
+#include <iostream>
+using std::cerr;
+using std::endl;
+using std::cout;
+#include <iomanip>        // std::get_time
+
+#include <sstream>
+#include <string>
+
+#include "RTOFSOb.h"
+#include "util.h"
+
+
+namespace rtofs
+{
+
+typedef char char12[12];
+typedef char char7[7];
+
+
+std::time_t
+    SecondsSinceReferenceTime(char12 time)
```

Comment:
We have had a problem with this process before, I don't know that the issue still exists but we need to make sure that this code doesn't make any shift defending on different machines-